salt/states/archive.py

```diff
@@ -174,6 +175,13 @@ def extracted(name,
 
         .. versionadded:: 2016.3.0
 
+   skip_verify:False
```

Comment:
Putting the default arg here is somewhat inconsistent compared to the other docs. Maybe move this to the body of the description?